Cylenium/Element.cs

```diff
@@ -95,6 +95,27 @@ public Element Check()
             return this;
         }
 
+        /// <summary>
+        /// Uncheck the checkbox or radio element.
+        /// </summary>
+        /// <returns>The current element.</returns>
+        public Element Uncheck()
+        {
+            var type = WebElement.GetAttribute("type");
+            if (type != "checkbox" && type != "radio")
+                throw new UnexpectedTagNameException($"Element.Check() expects a 'checkbox' or 'radio' element, but got {type}");
```

Comment:
I believe this is going to give you a problem, but I could be wrong. I think it's supposed to be a `OR` and not `AND`

Reply:
ooo yeah. good catch.